types/src/execution_result.rs

```diff
@@ -652,7 +633,7 @@ impl FromBytes for Transform {
 impl Distribution<Transform> for Standard {
     fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Transform {
         // TODO - include WriteDeployInfo and WriteTransfer as options
-        match rng.gen_range(0..14) {
+        match rng.gen_range(0..13) {
```

Comment:
I guess not for this PR, but it's worrying that we're not covering all variants here.  We use this in the bytesrepr roundtrip test, so only 13 of the 19 variants are being tested.